name.abuchen.portfolio.ui/src/name/abuchen/portfolio/ui/messages.properties

```diff
@@ -1516,6 +1516,8 @@ LabelRemoveDividends = Generate an additional withdrawal for dividend transactio
 
 LabelRemoveLogo = Remove logo
 
+LabelRemoveNotesInTransactions = Set notes (if available)
+
```

Comment:
This is flipping the meaning - the variable is called "remove notes", the text says "do not remove" notes. It works because at some later point the wording is flipped back.